guides/common/modules/con_offline-backup-and-online-backup.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="offline-backup-and-online-backup"]
+= Offline backup and online backup
+
+[role="_abstract"]
+To back up your {ProjectServer}, you can use an offline backup or an online backup.
```

Comment:
The abstract is new, the rest isn't. (Or actually, I also applied new markup to the text.)